perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -242,11 +194,12 @@
 DEFAULT = 'default'
 
 # This accounts for the overhead of running RemoteCommand() on a VM.
-MULTISTREAM_DELAY_PER_VM = 5.0
-# We wait this many seconds for each stream. Note that this is
-# multiplied by the number of streams per VM, not the total number of
-# streams.
-MULTISTREAM_DELAY_PER_STREAM = 0.1
+MULTISTREAM_DELAY_PER_VM = 10.0 * units.second
+# We wait this long for each stream. Note that this is multiplied by
+# the number of streams per VM, not the total number of streams.
+MULTISTREAM_DELAY_PER_STREAM = 15.0 * units.second
```

Comment:
Good point. I'd like to set these back to the old values for now - I didn't mean to put the new values in this PR. If we want to do this right in the long run, we'll have to benchmark how fast we can start processes on VMs and set these constants based on that, but I think that's outside the scope of this change.